test/integration/auth/svcaccttoken_test.go

```diff
@@ -77,7 +78,7 @@ func TestServiceAccountTokenCreate(t *testing.T) {
 	const iss = "https://foo.bar.example.com"
 	aud := authenticator.Audiences{"api"}
 
-	maxExpirationSeconds := int64(60 * 60)
+	maxExpirationSeconds := int64(60 * 60 * 2)
```

Comment:
Since I made expiration extension to be triggered by the special requested extension 60 * 60 + 7, if in this test we set max expiration seconds to 60 * 60, it would cap the requested expiration to 60 * 60, thus my special flow cannot be triggered by this test. -> I made it to 2 * 60 * 60